src/test_linked_list.py

```diff
@@ -155,13 +166,6 @@ def test_display_long(init_list, result):
     assert test_list.display() == result
 
 
-@pytest.mark.parametrize('init_list, result', TABLE_LENGTHS)
-def test_size(init_list, result):
```

Comment:
Why'd you kill this entire test? It's still valuable to test that the length of the list is appropriate for smaller sizes